projects/Vula/test.nix

```diff
@@ -1,21 +1,81 @@
 {
   sources,
-  pkgs,
+  lib,
   ...
-}: {
+}: let
+  inherit (lib) recursiveUpdate;
+in {
   name = "vula";
 
-  nodes = {
-    server = {config, ...}: {
+  nodes = let
+    common = {
       imports = [
         sources.modules.default
         sources.modules."services.vula"
       ];
+
       services.vula.enable = true;
+      services.vula.openFirewall = true;
+
+      services.vula.logLevel = "DEBUG";
```

Comment:
Probably this should be a lower level such as warn, error, or info but not debug.

Reply:
The default loglevel is `INFO`; we are setting `DEBUG` specifically for the test.